### ISSUE TITLE
`b2.utils.b2_url_decode` does not work in Python 3

### ISSUE THREAD

**Issue by bwbeach** [Backblaze/B2_Command_Line_Tool#466]
```
Traceback (most recent call last):
  File "/Users/brianb/sandbox/B2_Command_Line_Tool/test/test_utils.py", line 519, in test_it
    self.assertEquals(string, b2.utils.b2_url_decode(fully_encoded))
  File "/Users/brianb/sandbox/B2_Command_Line_Tool/b2/utils.py", line 81, in b2_url_decode
    return urllib.parse.unquote_plus(str(s)).decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by ppolewicz** [comment]
I think I fixed it yesterday on a branch

**Comment by bwbeach** [comment]
Cool!

**Comment by bwbeach** [comment]
This is working for me:

```
def b2_url_decode(s):
    """Decodes a Unicode string returned from B2 in an HTTP header.

    Returns a Python unicode string.
    """
    if six.PY2:
        return six.moves.urllib.parse.unquote_plus(str(s)).decode('utf-8')
    else:
        return six.moves.urllib.parse.unquote_plus(str(s))
```

**Comment by bwbeach** [comment]
Branch `encoding` has my fix, which is ugly because it checks the python version.

Looking forward to seeing @ppolewicz's fix.

**Comment by ppolewicz** [comment]
This was fixed by #476
